base-ci-resource.yaml

```diff
@@ -57,8 +57,9 @@ maven-build:
   script:
     - 'mvn clean versions:set -DnewVersion=$PACKAGE_VERSION'
     - 'mvn $MAVEN_CLI_OPTS package'
-  except:
-    - schedules
+  except:  
+    refs:
```

Comment:
This is the same functionally but needed if we also have an except for variables, so I standardized it